src/app/modules/consultations/consultation-profile/consultation-profile.graphql.ts

```diff
@@ -108,6 +108,25 @@ export const ConsultationProfile = gql`
 export const ConsultationProfileCurrentUser = gql`
   query consultationProfileCurrentUser($id: Int!) {
     consultationProfile(id: $id) {
+      anonymousResponses {
+        edges {
+          node {
+            id
+            consultation {
```

Comment:
@Nishanthh Is the Consultation field required here?

Reply:
@avikt not required bro, will remove it